style.css

```diff
@@ -530,7 +533,232 @@ ul {
+  width: 85%;
+  height: 75vh;
+}
+
+.modal.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+.modal-header {
+  display: flex;
+  padding: 10px 10px;
+  justify-content: space-between;
+}
+
+.modal-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.modal-list {
+  list-style: disc;
+  display: flex;
+  margin: 5px 0 20px 5px;
+}
+
+.close-modal-button {
+  background: transparent;
+  border: none;
+  color: #67798e;
+  font-size: 28px;
+}
+
+.modal-list li {
+  margin-right: 25px;
+  color: #7a869a;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.modal-list li:nth-child(1) {
+  list-style-type: none;
+  color: #344563;
+}
+
+.modal-content-detail {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  column-gap: 25px;
+}
+
+.modal-detial {
+  margin-top: 30px;
+  padding: 5px;
+}
+
+.modal-content-detail .modal-content-list .modal-language {
+  margin: 15px;
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+.modal-content-detail .modal-content-list .modal-language li {
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 10px;
+  width: fit-content;
+  padding: 1px 15px;
+  margin-right: 15px;
+  margin-top: 15px;
+}
+
+hr {
+  margin: 25px 20px 35px 20px;
+  border: 1px solid #ebecf0;
+  font-size: 3px;
+  width: 70%;
+}
+
+.model-links {
+  display: flex;
+  width: 20rem;
+  margin: 20px;
+  justify-content: space-evenly;
+}
+
+.modal-link-btns {
+  display: flex;
+  flex-wrap: wrap;
+  width: fit-content;
+  justify-content: center;
+  align-content: center;
+  align-items: end;
+  text-align: center;
+  background: #fff;
+  padding: 9px 20px;
+  border: 1px solid #6070ff;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 12px;
+  color: #6070ff;
+}
+
 @media (max-width: 768px) {
+  /* model-start */
+
+  .modal {
+    overflow-x: hidden;
+    width: 100%;
+    height: 100vh;
+    max-width: 80%;
+    padding: 15px;
+  }
+
+  .modal-content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .image-container {
+    height: 40vh;
+  }
+
+  .modal-image {
+    width: 100%;
+    height: 40vh;
+    align-self: center;
+  }
+
+  .modal-title {
+    font-size: 30px;
+    line-height: 32px;
+  }
+
+  .modal-list {
+    list-style: disc;
+    display: flex;
+    margin: 5px 0 10px 10px;
+  }
+
+  .close-modal-button {
+    font-size: 40px;
+  }
+
+  .modal-content-detail {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .modal-detial {
+    font-size: 15px;
+  }
+
+  hr {
+    margin: 25px 0;
+    width: 100%;
+  }
+
+  .modal.active {
+    transform: translate(-56%, -50%) scale(1);
+  }
+
+  .model-links {
+    width: 100%;
+    margin: 20px 5px;
+    justify-content: center;
+  }
+
+  .modal-link-btns {
+    padding: 9px 15px;
+    margin-right: 10px;
+  }
+
```

Comment:
- [ ] Kindly add the hover, disabled, pressed and enabled on the button that it very important for user experinece (UX)💪🏻🔥🔥
![image](https://user-images.githubusercontent.com/58553711/180223576-111d14ff-48fd-4c91-ac42-9e82f94bdbf3.png)

- [ ] As I can see here the width and the hright for the butttons are not the same as Figma kindly fix it


- Your design
![image](https://user-images.githubusercontent.com/58553711/180224689-bc19f77f-2906-4518-b82b-4edb57b98575.png)


- Figma design
![image](https://user-images.githubusercontent.com/58553711/180224912-302a49cd-09bc-4039-9971-7594c288297c.png)